.travis-ci.sh

```diff
@@ -24,7 +24,7 @@ opam_install_test_deps () {
     opam install \
          ocamlfind \
          ppxlib \
-         ppx_expect.v0.13.1 \
+         ppx_expect \
```

Comment:
```suggestion
         'ppx_expect>=v0.14' \
```

`ppx_expect.config_types` doesn't exist in previous versions